src/pages/Lend/Filter/SelectedRefinements.vue

```diff
@@ -1,38 +1,45 @@
 <template>
 	<div class="selected-refinements">
-		<ais-current-refinements
-			:transform-items="transformItems"
-		>
+		<ais-current-refinements :transform-items="transformItems">
 			<div slot-scope="{ items }">
-				<div class="filter-summary-container">
-					<div>{{ items.length }} Filters Applied</div>
-					<div v-if="isCollapsible" class="show-toggle-container">
-						<div v-if="isCollapsed" @click="handleClickShowMore">Show</div>
-						<div v-else @click="handleClickShowFewer">Hide</div>
+				<div class="row align-middle">
+					<div class="small-9 medium-10 columns">
```

Comment:
@foochris In the Designs for Mobile the selected filters take up the full width. this should be small-12